distro.json

```diff
@@ -243,5 +271,31 @@
         }
       }
     }
+  },
+  {
+    "name": "openapi_generator",
+    "platforms": {
+      "all": {
+        "universal": {
+          "url": "http://nxxm.indigenious.io/distro/v.0.0.15/openapi_generator/openapi-generator.zip",
+          "sha1": "ec7cbaed034d7aaeb2a0f22619a8927c08a74894",
+          "root": "openapi_generator",
+          "path": ["."]
+        }
+      }
+    }
+  },
+  {
+    "name": "openapi_generator_script",
+    "platforms": {
+      "all": {
+        "universal": {
+          "url": "http://nxxm.indigenious.io/distro/v.0.0.15/script/script_for_openapi_generator.zip",
```

Comment:
This should be stored on github, so that we can version it. Can you change that ?